src/theme/variables.scss

```diff
@@ -19,26 +24,120 @@ $tertiary: hsl(47, 100%, 92%);
+      "color-tertiary-contrast": contrast-colour.choose-contrast-color($tertiary),
+    );
+    $result: map.merge($result, $tertiary-map);
+  } @else {
+    // Auto generate a tertiary colour map from base colours
+  }
+  @return $result;
+}
+
+$colorPalette: getColorPalette($color-primary, $color-secondary);
+
+@mixin ionColorPalette {
+  --ion-color-primary: #{map-get($colorPalette, "color-primary")};
+  --ion-color-primary-light: #{map-get($colorPalette, "color-primary-light")};
+  --ion-color-primary-lighter: #{map-get($colorPalette, "color-primary-lighter")};
+  --ion-color-primary-contrast: #{map-get($colorPalette, "color-primary-contrast")};
+
+  --ion-color-secondary: #{map-get($colorPalette, "color-secondary")};
+  --ion-color-secondary-light: #{map-get($colorPalette, "color-secondary-light")};
+  --ion-color-secondary-lighter: #{map-get($colorPalette, "color-secondary-lighter")};
+  --ion-color-secondary-contrast: #{map-get($colorPalette, "color-secondary-contrast")};
+
+  --ion-color-tertiary: #{map-get($colorPalette, "color-tertiary")};
+  --ion-color-tertiary-light: #{map-get($colorPalette, "color-tertiary-light")};
+  --ion-color-tertiary-lighter: #{map-get($colorPalette, "color-tertiary-lighter")};
+  --ion-color-tertiary-contrast: #{map-get($colorPalette, "color-tertiary-contrast")};
+
+  //  GRADIENTS
+  // primary-gradient
+  --ion-gradient-primary-start: #{map-get($colorPalette, "gradient-primary-start")};
+  --ion-gradient-primary-end: #{map-get($colorPalette, "gradient-primary-end")};
+  // secondary-gradient
+  --ion-gradient-secondary-start: #{map-get($colorPalette, "gradient-secondary-start")};
+  --ion-gradient-secondary-end: #{map-get($colorPalette, "gradient-secondary-end")};
+}
+
+.theme-default {
+  @include ionColorPalette;
+}
+
 /** Ionic CSS Variables **/
 :root {
-  --ion-main-bg-passive: #{$quaternary};
-  --ion-main-bg-active: #{$tertiary};
+  //   --ion-main-bg-passive: #{$quaternary};
+  //   --ion-main-bg-active: #{$tertiary};
+
+  // Buttons
+  --ion-btn-primary: linear-gradient(
+    #{$vertical},
+    #{map-get($colorPalette, "gradient-primary-start")} 0.43%,
+    #{map-get($colorPalette, "gradient-primary-end")} 74.07%
+  );
+  --ion-btn-secondary: linear-gradient(
+    #{$vertical},
+    #{map-get($colorPalette, "gradient-secondary-start")} 28.12%,
+    #{map-get($colorPalette, "gradient-secondary-end")} 100%
+  );
+  --ion-btn-control-bg: #{color.adjust($primary, $hue: 3.6deg, $saturation: 3%, $lightness: 25%)};
+
+  //  Borders
+  --ion-border-standard: 2px solid #{map-get($colorPalette, "color-primary")};
+  --ion-border-thin-standard: 1px solid #{map-get($colorPalette, "color-primary")};
+  --ion-border-color-secondary: 2px solid #{map-get($colorPalette, "color-secondary")};
+  --ion-border-light: 1px solid #{$light};
+  --ion-border-light-thicker: 2px solid #{$light};
+  --border-dashed: 2px dashed #{map-get($colorPalette, "color-primary")};
 
-  // Margins //
+  //   // Margins //
```

Comment:
I don't understand the double quoting, in some editors this will show as a crossed-out comment.

Reply:
Ah yeh, just an artefact of commenting out a whole block (including comments) and then uncommenting some of it